charts/px-backup/templates/pxcentral-pre-install-hook.yaml

```diff
@@ -0,0 +1,76 @@
+{{/* Setting defaults if they are omitted. */}}
```

Comment:
I'm still not sure why this needs to be a pre-install hook. Can this be an init container instead for whichever module requires it?

Reply:
Currently, in pre-install hook we have validations of all input values like storage class, custom registry details, external oidc details. We are validating this inputs in pre-install hooks and if only it succeeds then create all the remaining components else will not create any other object.

Reply:
You can do that in the helm template itself. Example here: https://github.com/helm/helm/issues/3497

Reply:
I tried to use kubectl commands in the template itself to verify given storage class or image pull secrets exists or not, but it's not working.